data/data-pipeline/data_pipeline/etl/sources/tree_equity_score/etl.py

```diff
@@ -87,4 +87,42 @@ def load(self) -> None:
         logger.info("Saving Tree Equity Score GeoJSON")
         # write nationwide csv
         self.CSV_PATH.mkdir(parents=True, exist_ok=True)
-        self.df.to_file(self.CSV_PATH / "tes_conus.geojson", driver="GeoJSON")
+        # here I remove the geometry
+        self.df = self.df[
+            [
+                "geoid",
```

Comment:
since we're looking at tracts in this dataset, can you go ahead and map this field to the tract field constant (`GEOID_TRACT_FIELD`) from the `field_names` module? this will help make it easier to join with other datasets later because the join column will be named the same.

Reply:
A fantastic idea!

Reply:
We no longer have a reference of `GEOID_TRACT_FIELD` in field names. However, what I do see consistently is `GEOID_TRACT_FIELD_NAME` as an instance variable of any subclass of `ExtractTransformLoad`. That is what is added now.